src/protocols/rdp/guac_rdpdr/rdpdr_service.h

```diff
@@ -74,23 +74,43 @@ struct guac_rdpdr_device {
     int device_id;
 
     /**
-     * An arbitrary device name, used for logging purposes only.
+     * Device name, used for logging and for passthrough to the
+     * server.
      */
     const char* device_name;
 
     /**
-     * Handler which will be called when the RDPDR plugin is forming the client
-     * device announce list.
+     * The length of the device name, used for generating stream.
      */
-    guac_rdpdr_device_announce_handler* announce_handler;
+    int device_name_len;
+
+    /**
+     * The type of RDPDR device that this represents.
+     */
+    uint32_t device_type;
+
+    /**
+     * The DOS name of the device.  Max 8 bytes, including terminator.
+     */
+    char *dos_name;
```

Comment:
Like `device_name`, this should probably also be `const char*`, unless you think they should both be `char*`.

Reply:
`const` is fine with me

Reply:
const'd.